extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -405,10 +405,21 @@
 		_t.image = parentNode.querySelector("img");
 		_t.updateInterval = parseInt(parentNode.getAttribute("data-update-interval"), 10);
 
-		_t.url = _t.image.getAttribute("src").replace(/&t=\d+/g, "");
```

Comment:
@resetnow : if you really think that it is better to include the refresh attribute in this new data atribute on render servlet side and then suppress it here on client side, I can do it.
For me, that is better like I did, but no problem to change it.